CMakeLists.txt

```diff
@@ -1830,7 +1830,13 @@ MYSQL_CHECK_LDAP()
 MYSQL_CHECK_LDAP_DLLS()
 
 # Percona LDAP Simple / LDAP SASL authentication plugins
-OPTION(WITH_PERCONA_AUTHENTICATION_LDAP "Build with Percona LDAP Simple / LDAP SASL authentication plugins" ON)
+# Problem with  MacOS system libraries and complains for compilation error ldap_set_urllist_proc
+IF(APPLE)
+	OPTION(WITH_PERCONA_AUTHENTICATION_LDAP_DEFAULT OFF)
```

Comment:
This should be a SET command, not an option.

Reply:
fixed